.github/ISSUE_TEMPLATE/product-update-form.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: Update a Product
+description: Update information about a product in the database
+title: "[Product Update]: <vendor> - <product>"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        For questions about choice for status, please see the information below.
+
+        - Unknown - Status unknown. Default choice.
+        - Affected - Reported to be affected by CVE-2021-44228.
+        - Not Affected - Reported to NOT be affected by CVE-2021-44228 and no
+          further action necessary.
+        - Fixed - Patch and/or mitigations available (see provided links).
+        - Under Investigation - Vendor investigating status.
+  - type: textarea
```

Comment:
The submission and update forms are similar, with the update form less strict, with some validations skipped, and the `product-vendor` and `product-name` inputs skipped as well. Would it technically help to combine the two forms into one `[Product information]` form, with an additional, required `submission-type` input settable to "Product submission" or "Product update" added?

Reply:
When adding a new product there is important information that is required, but there is no need to force a submitter to fill out information that isn't changing when submitting a request to update a product.

Reply:
> When adding a new product there is important information that is required, but there is no need to force a submitter to fill out information that isn't changing when submitting a request to update a product.

I agree with this wholeheartedly.  The less information that users submit, the fewer opportunities there are for errors.